### DISCUSSION TITLE
URL link to comment is rendered as raw URL if the link is copied in Files tab

### DISCUSSION THREAD

**Discussion by nozomuikuta** [community/community#110950] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Thank you for managing GitHub.
I rely on your great service everyday, and my product can't be possible without yours.

---

Today, I found a space to improve in PR page.

In `https://github.com/<namespace>/<repo>/pull/<number>` (i.e. Conversation tab), I can copy URL to a specific comment.
The copied URL is like `https://github.com/<namespace>/<repo>/pull/<number>#discussion_r1234567890`.

On the other hand, if I copy URL to the same comment in Files tab, the copied URL becomes `https://github.com/<namespace>/<repo>/pull/<number>/files#r1234567890`

Now, if I paste the URL in comment textarea, while the former is rendered as `#<pr_number> (comment)`, the latter is rendered as raw URL.

Since the 2 URLs eventually point to the same comment, it would be great if both are rendered as formatted link and support popover.

**Comment by MasterInQuestion** [comment]
It would be better not adding such meddling: for links may as well representing themselves.
    E.g. [https://github<i>.</i>com/MasterInQuestion/talk/discussions/2#discussioncomment-7582726](https://github.com/MasterInQuestion/talk/discussions/2#discussioncomment-7582726)

    Such change may cause significant backward-compatibility issue as well. (inadvertently broken existing content)
